samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -155,6 +155,8 @@ const StorybookDecorator = (story, config = {}) => {
 		groupId: 'Development'
 	};
 
+	const componentName = config.kind.replace(/^([^/]+)\//, '');
```

Comment:
I think we can also use regExp in this way.
```suggestion
	const componentName = config.kind.replace(/^\w+\//, '');
```